pkg/worker/pool.go

```diff
@@ -158,6 +159,45 @@ func filterWorkers(ws []Worker, f func(w Worker) bool) []Worker {
 	return newWs
 }
 
+// GetWorkers - return a slice of all workers matching the input options.
+// useful for retrieving a list of all topic subscribers (for example)
+func (p *ProcessPool) GetWorkers(opts *GetWorkerOptions) []Worker {
+	p.workerLock.Lock()
+	defer p.workerLock.Unlock()
+
+	workers := make([]Worker, 0)
+
+	if opts.Http != nil {
+		ws := p.getHttpWorkers()
+
+		if opts.Filter != nil {
+			ws = filterWorkers(ws, opts.Filter)
+		}
```

Comment:
Because we don't have a case where we need to call the actual hander for a single worker we could move this to the end of the function I think

Reply:
done